### DISCUSSION TITLE
How to add featured images to pages other than homepage?

### DISCUSSION THREAD

**Discussion by ooexiaoo** [nunocoracao/blowfish#292] [General]
I'm confused about how to add images to pages other than the homepage and I'm not talking about blog posts here.
I'm talking about category pages or any other page I've created for example a page with all my art blog posts under that page "ART".

I see background images for different pages on the demo blog of blowfish, but I can't seem to find the settings as to how to achieve it by myself.

**Comment by nunocoracao** [comment]
Hello @ooexiaoo take a look at these docs:

https://nunocoracao.github.io/blowfish/docs/thumbnails/

https://nunocoracao.github.io/blowfish/samples/thumbnail_sample/

It should help

**Reply by ooexiaoo** [reply]
Thanks!
Actually, I found the answer, I had to change `params.toml` >`[list]` to `showHero = true` to see hero images in other category pages that I've created.

**Reply by danraymond** [reply]
Hello, same issue. having set showHero=true, where do I specify the image? In the frontmatter of the "ART" page, and how to do please?

**Reply by danraymond** [reply]
Okay worked out that the  feature image works for list pages if they are  too but how to do for the "tag" and "series" list pages as they don;t exist till you call them?

**Reply by nunocoracao** [reply]
They do exist and you need to create specific folders to place the images. Check Blowfish repo for examples and read Hugo's docs.

**Comment by danraymond** [comment]
Hello,

Thank you. I did  but initially didn't fully understand! Now I have tried
more times. It was so simple I just didn't pick up that the feature image
would also work for list pages (duh!)

After trying so many themes yours was the only one that had all the
facilities I needed.

Just a loose question: if I wanted a "topic" system just like the series
system would this work:
1. create the taxonomy "Topics"
2. duplicate series.html and series_order.html and rename to topic.html and
topic_order.html

Would that be enough or is there code elsewhere to  manipulate?

Thank you

Dan

**Reply by nunocoracao** [reply]
@danraymond check in blowfish how I created the taxonomy authors. Also check Hugo's docs for the details on how to do it.

**Comment by danraymond** [comment]
Hello Nuno,

Thank you, I will.

**Comment by danraymond** [comment]
HI Nuno,

I have nearly nailed the topics functionality similar to the "series"
function in Blowfish. A small question: Where does the statement text "This
article is part of a series" come from in the code? I can see this: {{
index .Params.topics 0 }} - {{ i18n "article.part_of_series" }} but
where does .part_of_series get the text string "This article is part of a
series"

Thank you

Dan

**Comment by danraymond** [comment]
I finally sorted it!

Now  to see if I can do a hierarchy of series within a topic!!

Thank you for your help.

Regards
Dan
